src/components/PlantsPrimaryPage/index.tsx

```diff
@@ -17,7 +17,8 @@ export type PlantsPrimaryPageProps = {
   setPlantsSitePreferences: (preferences: Record<string, unknown>) => void;
   allowAllAsSiteSelection?: boolean; // whether to support 'All' as a planting site selection
   isEmptyState?: boolean; // optional boolean to indicate this is an empty state view
-  onPlantingSites?: (plantingSites: PlantingSite[]) => void; // optional callback to pass planting sites list on fetch
+  // this is to allow redux based components to pass in already selected data
+  plantingSitesData?: PlantingSite[];
```

Comment:
this is so we don't re-request the same data (redux vs non-redux client components)